event_processor/event_processor/base/spider_base.py

```diff
@@ -42,4 +43,10 @@ def create_time_data(self, **kwargs):
             if len(value) != count:
                 raise ValueError(f'{self.organization}: Time selectors returned data of differing lengths')
         return [{key: value[i] for key, value in kwargs.items()} for i in range(count)]
+
+    def normalize_whitespace(self, s):
+        """Convert all types of whitespace, including new lines, into a normal space"""
+        if s == None or s == "":
+            return ""
+        return (" ".join(list(filter(lambda x: re.fullmatch(r'[ ]*', x) == None, s.replace('\n', ' ').replace('\t', ' ').replace('\r', ' ').split(' '))))).strip()
```

Comment:
I think this last line can be replaced with `return re.sub('/s+', ' ', s).strip()`
```suggestion
        return re.sub('/s+', ' ', s).strip() 
```